src/poker/game/ActionHandler.java

```diff
@@ -0,0 +1,175 @@
+        this.gameState = gameState;
+        this.result = result;
+        this.deck = deck;
+        this.player1 = player1;
+        this.player2 = player2;
+        this.equityEvaluator = equityEvaluator;
+    }
+
+    public HandleActionResult handleAction(PossibleAction lastAction, Player activePlayer, Player nonActivePlayer) {
+        HandleActionResult result = new HandleActionResult();
+        if (lastAction == PossibleAction.LIMP) {
+            hanldeLimp(activePlayer);
+        } else if (lastAction == PossibleAction.RAISE) {
+            handleRaisePlayer(activePlayer, nonActivePlayer);
+        } else if (lastAction == PossibleAction.BET) {
+            handleBetPlayer(activePlayer);
+        } else if (lastAction == PossibleAction.CALL) {
+            boolean isAllin = handleCall(activePlayer, nonActivePlayer);
+            result.setAllIn(isAllin);
+        } else if (lastAction == PossibleAction.CHECK) {
+            System.out.println(activePlayer.getNamePlayer() + " checks");
+        } else {
+            result.setFold(true);
+            handleFold(activePlayer, nonActivePlayer);
+        }
+        return result;
+    }
+
+    public void handleHandStart(Player activePlayer, Player nonActivePlayer){
+        if (activePlayer.getStackPlayer() < 100) {
+            activePlayer.increaseStackPlyer(100 - activePlayer.getStackPlayer());
+        }
+        if (nonActivePlayer.getStackPlayer() < 100) {
+            nonActivePlayer.increaseStackPlyer(100 - nonActivePlayer.getStackPlayer());
+        }
+        updatePotAndStackAmount(gameState.bigBlind, nonActivePlayer);
+        updatePotAndStackAmount(gameState.smallBlind, activePlayer);
+        activePlayer.increasePlayermoneyOnStreet(gameState.smallBlind);
+        nonActivePlayer.increasePlayermoneyOnStreet(gameState.bigBlind);
+        GamePlayUtils.printPreFlopDetails(nonActivePlayer, activePlayer, result, gameState);
+    }
+    public void handleNextHand() {
+        for (int i = 0; i < 10; i++) {
+            System.out.println();
+        }
+        try {
+            Thread.sleep(4000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+        this.result = deck.shuffleDeckAndGetHandsAndBoard();
+        this.gameState.setGamePhase(GamePhase.PREFLOP);
+        player1.setHandPlayer(result.getHand1());
+        player2.setHandPlayer(result.getHand2());
+    }
+
+
+    public boolean handleCall(Player activePlayer, Player nonActivePlayer) {
+        updatePotAndStackAmount(GamePlayUtils.amountToCall(nonActivePlayer, activePlayer), activePlayer);
+        return activePlayer.getStackPlayer() == 0 || nonActivePlayer.getStackPlayer() == 0;
+    }
+
+
+    public void handleNextStreet(GameState gameState, Player activePlayer, Player nonActivePlayer) {
+        if (gameState.getGamePhase() == GamePhase.PREFLOP) {
+            GamePlay.resetMaxValueRaises(activePlayer,nonActivePlayer);
```

Comment:
GamePlay.resetMaxValueRaises(activePlayer,nonActivePlayer); dales do kazdego ifa no XD daj to przed ifem na samej gorze funkcji to nie bedzie trzeba tego 4 razy pisac tylko 1 raz